### PR TITLE
Switch from DSE Graph to Neo4j

### PR DESCRIPTION
**Pull request by sproberts92**
All DSE Graph functionality is removed from hubble-core and replaced with Neo4j functionality.

Cluster info is packaged into a json document which is sent to Neo4j and optionally dumped to disk.

### CODE REVIEW

File: hubble-core/src/main/scala/team/supernova/graph/output/GraphDumper.scala [hunk 1]
```diff
@@ -1,30 +1,52 @@
...
-      keyspaceInfoMap.getOrElse("Keyspace Summary", Map[String, KeyspaceInfo]()).persistToGraph("Keyspace Summary", session)
+  def dumpClusterInfoSet(session: Session, jsonDumpDir: String, clusterInfoSet: Set[ClusterInfo]): Unit = {
+    log.info("Received clusterinfo data, dumping to json.")
+    val jsonData = ClusterInfoSetHelper.ClusterInfoSetFunctions(clusterInfoSet).toJson
+
+    if(!jsonDumpDir.isEmpty){
+      /* Json will dumped with a timestamped filename. */
+      val formattedDate = new SimpleDateFormat("yyyyMMddHHmmss").format(new Date())
+
+      /* To do: Read dump directory from config. */
```
Comment (jkazakovs) [inline hubble-core/src/main/scala/team/supernova/graph/output/GraphDumper.scala]: Use standard TODO notation. IDE will interpret it properly and mark as todo item.

File: hubble-core/src/main/scala/team/supernova/graph/output/GraphDumper.scala [hunk 2]
```diff
@@ -1,30 +1,52 @@
...
     }
 
-}
-  def dumpClusterInfoSet(session: DseSession, clusterInfoSet: Set[ClusterInfo]): Unit = {
-    log.info("Received clusterinfo data, sending to graph")
-    clusterInfoSet.toGraph(session)
-    log.info("Finished sending data to graph")
+    log.info("Starting to send data to graph.")
+
+    val cypherResouce = getClass.getResource("/neo4j/load_hubble_json.cql")
```
Comment (jkazakovs) [inline hubble-core/src/main/scala/team/supernova/graph/output/GraphDumper.scala]: typo, use cypherResource instead

File: project/Dependencies.scala [hunk 3]
```diff
@@ -93,6 +94,7 @@ object Dependencies {
   val hubbleGraphDependencies = Seq(
     Cassandra.DseDriver,
     Cassandra.DseGraph,
```
Comment (jkazakovs) [inline project/Dependencies.scala]: this dependency is probably not needed

Comment (sproberts92) [inline project/Dependencies.scala]: For hubble-core that is correct, however it seems there are still some dse graph components outside core so I left it for now.

File: hubble-core/src/main/scala/team/supernova/graph/output/GraphDumper.scala (hunk 1 above)
Comment (sproberts92) [inline hubble-core/src/main/scala/team/supernova/graph/output/GraphDumper.scala]: That should be removed as I did it, but what is the standard notation?

Comment (jkazakovs) [inline hubble-core/src/main/scala/team/supernova/graph/output/GraphDumper.scala]: TODO

File: project/Dependencies.scala (hunk 3 above)
Comment (jkazakovs) [inline project/Dependencies.scala]: as I understand it correctly, the initial goal was to refactor DSE API to Neo4j one, at least from hubble-core component. if you remove DSE dependency and you code does not compile this implies that the refactoring is not complete.

Comment (jkazakovs) [inline project/Dependencies.scala]: I think this can be simply solved by removing dependency on hubble-graph from hubble-core component

**Pull request merged**